src/app.js

```diff
@@ -1 +1,25 @@
 'use strict';
+
+const fs = require('fs');
+
+const copyFile = (copyFrom, copyTo) => {
+  if (copyFrom === copyTo) {
+    throw new Error('The pathes are the same');
```

Comment:
not sure if there's a word in english such as pathes, it's better to use paths 😉